IDENTITY_USE_CASES.md

```diff
@@ -0,0 +1,98 @@
+# Identity Use Cases in OpenSearch
+- [Identity Use Cases in OpenSearch](#identity-use-cases-in-opensearch)
+  - [Non-use compatibility](#non-use-compatibility)
+    - [Scenario 1:](#scenario-1)
+  - [Identity features enabled](#identity-features-enabled)
+    - [Scenario 2:](#scenario-2)
+    - [Scenario 3:](#scenario-3)
+  - [Using Admin account](#using-admin-account)
+    - [Scenario 4:](#scenario-4)
+    - [Scenario 5:](#scenario-5)
+  - [Password changes](#password-changes)
+    - [Scenario 6:](#scenario-6)
+    - [Scenario 7:](#scenario-7)
+    - [Scenario 8:](#scenario-8)
+  - [Authenticate as new user](#authenticate-as-new-user)
+    - [Scenario 9:](#scenario-9)
+    - [Scenario 10:](#scenario-10)
+    - [Scenario 11:](#scenario-11)
+  - [Multi-node Cluster](#multi-node-cluster)
+    - [Scenario XXX:](#scenario-xxx)
+
+## Non-use compatibility
+
+    Given: Start an OpenSearch cluster with Identity experiment enabled and not configured
+
+### Scenario 1:
+
+The cluster runs with the Security plugin. No Security plugin scenarios are impacted.
+
+*Technical*: Identity uses the NoopAuthenticationManager
+
+## Identity features enabled
+
+    Given: Start an OpenSearch cluster with Identity experiment enabled, internal identity is enabled in the configuration
+
+### Scenario 2:
+
+By reading the opensearch.log / console output a random admin password is shown.
+
+### Scenario 3:
+
+All REST API activity returns 403 without passing authentication information in the request
+
+## Using Admin account
+
+    Given: The admin password is known
+
+### Scenario 4:
+
+Using http basic auth it is possible to make any REST API request, no errors related to authentication are seen.
```

Comment:
Should we add a note here stating `new request requires new authentication`?

Reply:
I don't think we need that kind of detail, unless there were scenarios where authentication was saved (we shouldn't have any, you always need to provide your creds/token to identify the requestor)